src/DataFixtures/FeedFixtures.php

```diff
@@ -14,7 +14,7 @@ public function load(ObjectManager $manager): void
         $feed = new Feed();
         $config = [
             'type' => 'json',
-            'url' => 'https://www.aros.dk/umbraco/api/events/feed/?culture=da',
+            'url' => '/app/src/DataFixtures/feeds/aros.json',
```

Comment:
I think we should remove `/app/` from the URL. `/app/` depends on how this is run, e.g. with (our) `docker compose`, and loading fixtures should work generally.

The current working directory when loading fixtures should be “project root”, so removing `/app/` from the URL should work (I haven't tried it).

```suggestion
            'url' => 'src/DataFixtures/feeds/aros.json',
```

Reply:
Fixed and testet that it works with relative paths.